driver/wnbd_dispatch.c

```diff
@@ -83,7 +68,7 @@ NTSTATUS LockUsermodeBuffer(
 
 NTSTATUS WnbdDispatchRequest(
     PIRP Irp,
-    PSCSI_DEVICE_INFORMATION DeviceInfo,
+    PWNBD_SCSI_DEVICE Device,
     PWNBD_IOCTL_FETCH_REQ_COMMAND Command)
 {
     // TODO: reject request when using NBD.
```

Comment:
Can these be dropped now?

Reply:
Definitely, I forgot to remove the TODO. Thanks!